test/multiselect.js

```diff
@@ -0,0 +1,125 @@
+const child_process = require('child_process');
```

Comment:
To get more confidence than the existing tests provide, I created a test that spawns a child process and interacts with it as a user would.

Reply:
I'm looking into how to test the lib for version 3. If you have more ideas for how to do testing, please let me know.

Reply:
I think this file is a good test approach, and it can be expanded to the rest of the lib.